.github/actions/github-pull-request-action/README.md

```diff
@@ -0,0 +1,65 @@
+<!--
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+-->
+
+# #️⃣ GitHub Pull Request
```

Comment:
Rename the action to "Github Pull Request Info", we are'nt creating PR's by only deriving some info about the PR's here.